src/indexers/vault-balances-indexer.ts

```diff
@@ -12,12 +12,10 @@ import { getCachedVault } from '../vaults/vaults.utils';
 import { getLpTokenBalances } from './indexer.utils';
 
 export async function refreshVaultBalances() {
-  await Promise.all(
-    SUPPORTED_CHAINS.flatMap(async (c) => {
-      const vaults = await c.vaults.all();
-      return vaults.map(async (v) => updateVaultTokenBalances(c, v));
-    }),
-  );
+  for (const chain of SUPPORTED_CHAINS) {
```

Comment:
wouldn't you still want a promise.all() around the chain forloop?

Reply:
no this should be fine as is